app/helpers/auth0_helper.rb

```diff
@@ -15,7 +15,8 @@ def login_path
   # Is the user signed in?
   # @return [Boolean]
   def user_signed_in?
-    session.try(:[],:user_id).present?
+    request.path != '/'
```

Comment:
Is this a temporary solution?

Reply:
this is just to separate out the styling from the login functionality. it will change as soon as the auth0 pr is raised